composer.json

```diff
@@ -25,6 +25,7 @@
     "require": {
         "php": "^7.0",
         "symfony/framework-bundle": "^3.1",
+        "symfony/contracts": "^1.0",
```

Comment:
Since the phpcs update commited the composer.lock, I need this for the tests of ItemInterface and the $cacheItem->tag() method.